components/Modal/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Swal from 'sweetalert2'
+
+export default function Modal(str: string, callback: any) {
```

Comment:
maybe something like:
```suggestion
export default function Modal(str: string, confirmButtonText: any) {
```
or similar